pkgs/by-name/sm/smartcat/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+
+, openssl
+, pkg-config
+, libX11
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "smartcat";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "efugier";
+    repo = "smartcat";
+    rev = "refs/tags/v${version}";
```

```suggestion
    rev = "refs/tags/${version}";
```

the release was titled v1.3.0, but the tag is actually 1.3.0, oops